src/javascript/app_2/Modules/Page404/Components/Page404.jsx

```diff
@@ -0,0 +1,27 @@
+import React          from 'react';
+import { routes }     from 'Constants/index';
+import { localize }   from '_common/localize';
+import { Icon404 }    from './Icon404.jsx';
+import { ButtonLink } from '../../../App/Components/Routes';
```

Comment:
@zakibinary 
using alias path makes the maintenance more easier
```suggestion
import { ButtonLink } from 'App/Components/Routes';
```